### ISSUE TITLE
Tax assessment data via opendata API

### ISSUE THREAD

**Issue by NealHumphrey** [codefordc/housing-insights#198]
Opendata.dc.gov provides [tax assessment data](http://opendata.dc.gov/datasets/integrated-tax-system-public-extract-facts) of all properties in DC via an API.

- [ ] Create a TaxApiConn class based on the MarApiConn class currently in the code. Use this to add raw data files to /data/raw/tax_assessment/opendata/YYYYMMDD when the python/cmd/data.py script is run with the appropriate arguments. Write a demo command to run at the command line to create this file with the appropriate datestamp.
(pull request after step 1; someone else can do part 2 if desired)
- [ ] Add this file to the manifest.csv using the instructions on [adding a new dataset](http://housinginsights.org/resources/adding-data-sources.html).

Future: automatically add new files to the pipeline whenever this update script is run, but we wait to do this until we have created an appropriate structure for all API scripts.

**Comment by NealHumphrey** [comment]
Note on integrating this data from open.data@dc.gov:

EXTRACTDATE field says when this was last updated, looks to be roughly monthly.

**Comment by NealHumphrey** [comment]
Hey @eng1nerd have you been able to do any work on this issue? Let me know where things stand - and if you're busy with the new job we can also see if someone's able to take it over!

**Comment by eng1nerd** [comment]
@NealHumphrey I'm sorry for the delay. I will try to finish it by tomorrow 6pm. If something will not be right or it will take me much longer, then some other person should probably step in.

**Comment by NealHumphrey** [comment]
Latest status:

- @eng1nerd 's code has been merged into the codefordc repository under the branch name `198-add-tax-data`; should pick up code from there. 
- Current TaxApiConn class uses the 'GeoService` api. Instead we should use the 'GeoJSON' api url, but swap it out for .csv as noted on the opendata documentation (bottom of page): http://opendata.dc.gov/pages/using-apis . This will resolve the issue that the number of rows is limited in the geojson api (and also make it easier for us to parse since it'll be in the format we want anyways).
